test/scripts/create-proxy.test.js

```diff
@@ -56,6 +57,11 @@ contract('create-proxy command', function([_, owner]) {
     await createProxy({ contractAlias: "NotExists", packageFileName, network, txParams }).should.be.rejectedWith(/not found/);
   });
 
+  it('should refuse to create a proxy for a lib project', async function() {
+    editJson(packageFileName, p => { p.lib = true; });
+    await createProxy({ contractAlias, packageFileName, network, txParams }).should.be.rejectedWith(/lib/);
+  });
+
```

Comment:
why do we want to prevent lib developers to instantiate proxies?

Reply:
Discussed offline